addons/packages/ako-operator/1.5.0/bundle/config/values.yaml

```diff
@@ -18,15 +18,20 @@ akoOperator:
     avi_password: ""
     avi_cloud_name: "Default-Cloud"
     avi_service_engine_group: "Default-Group"
+    avi_management_cluster_service_engine_group: "MC-SEG"
     avi_data_network: ""
     avi_data_network_cidr: ""
+    avi_cp_network: ""
+    avi_cp_network_cidr: ""
     avi_ca_data_b64: ""
     avi_labels: '""'
     avi_cni_plugin: antrea
     avi_disable_static_route_sync: false
     avi_control_plane_ha_provider: false
     avi_management_cluster_vip_network_name: ""
     avi_management_cluster_vip_network_cidr: ""
+    avi_management_cluster_cp_vip_network_name: ""
```

Comment:
`cp` is not clear about what it actually meaning, we can change it to `control_plane`